integrations/docker/images/chip-build-bl602/Dockerfile

```diff
@@ -0,0 +1,10 @@
+ARG VERSION=latest
+FROM connectedhomeip/chip-build:${VERSION}
+
+RUN set -x \
+    && apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -fy --no-install-recommends \
+    git=1:2.25.1-1ubuntu3.4 \
```

Comment:
This seems very close to just chip-build. Why is a separate dockerfile needed?

Reply:
Thanks for your review, we see there are different dockerfile and docker build/image for different platfrom, and considering the further extensions to integrate more bl602 specific tools/scripts, so we decide to use a separate dockerfile.

Reply:
changed to use chip-build.